life_expectancy/tests/test_loaders.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for the loaders module"""
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+import pandas as pd 
+
+from life_expectancy.loaders import save_data, load_data
+
+
+BASE_DIR = Path(__file__).parent.parent / "data"
+FIXTURES_DIR = Path(__file__).parent / "fixtures"
+# print(BASE_DIR)
+
+
+@patch("life_expectancy.loaders.pd.read_csv")
+def test_load_data(mock_read_csv, eu_life_expectancy_expected) -> None:
+
+    """testing if the load_data is called with the necessary arguments"""
+    mock_read_csv.return_value = eu_life_expectancy_expected
+    input = "pt_life_expectancy_expected_test.csv"
+    actual = load_data(input)
+    mock_read_csv.assert_called_once_with(BASE_DIR / input, delimiter="\t")
+    pd.testing.assert_frame_equal(actual, eu_life_expectancy_expected)
```

Comment:
The fixture `eu_life_expectancy_expected` and loaded data `actual` coming from the same function `load_data` so the assert_frame_equal will always be correct in this situation, thus I think it is a bit redundant to be called like this. Maybe you can create a simple dataframe using from_dict and a simple csv fixture following that to test.